diesel_cli/tests/migration_run.rs

```diff
@@ -346,3 +346,29 @@ file = "src/my_schema.rs"
     );
     assert!(p.has_file("src/my_schema.rs"));
 }
+
+#[test]
+fn migrations_can_be_run_with_no_config_file() {
+    let p = project("migration_run").folder("migrations").build();
+    let db = database(&p.database_url());
+
+    p.command("database setup").run();
```

Comment:
CI says

> Failed to open connection to postgres://postgres@localhost/diesel_migration_run: BadConnection("FATAL:  database "diesel_migration_run" does not exist

which leads me to ask: Isn't it just "setup" here? Also, `run` doesn't seem to assert the command exited with zero, so it can't catch wrong calls.

Reply:
`diesel setup` will create a config file

Reply:
ah that's true, damn

Reply:
I just forgot to change the project name in this test. It's a race condition, not an actual bug.